specification/search/data-plane/Microsoft.Azure.Search.Data/preview/2017-11-11-preview/searchindex.json

```diff
@@ -64,6 +64,325 @@
         }
       }
     },
+    "/docs": {
+      "get": {
+        "tags": [
+          "Search"
```

Comment:
Only $count sets it to `DocumentsProxy`, autocomplete doesnt.  From docs on tags ([https://swagger.io/docs/specification/2-0/grouping-operations-with-tags/](https://swagger.io/docs/specification/2-0/grouping-operations-with-tags/)) it seems tags are used only in grouping items in rendering in the Swagger UI (which I've used and found helpful. [https://editor.swagger.io/](https://editor.swagger.io/)) I propose we tag by operation types, setting all to DocumentsProxy doesn't buy us anything; we might as well remove tags if all ops have the same tag. 

---
In reply to: [208096878](https://github.com/Azure/azure-rest-api-specs/pull/3578#discussion_r208096878) [](ancestors = 208096878)

Reply:
The tag is only the same because all operations in this spec belong to the same group. Take a look at searchservice.json -- there is a different tag for each resource, which makes sense to me.

Although if tags are no longer used by AutoRest for codegen, I propose we use "Documents" instead of "DocumentsProxy"

Reply:
Okay. I like "Documents" better too